cmd/libs/go2idl/client-gen/generators/generator_for_type.go

```diff
@@ -66,7 +66,7 @@ func hasStatus(t *types.Type) bool {
 func (g *genClientForType) GenerateType(c *generator.Context, t *types.Type, w io.Writer) error {
 	sw := generator.NewSnippetWriter(w, c, "$", "$")
 	pkg := filepath.Base(t.Name.Package)
-	namespaced := !(types.ExtractCommentTags("+", t.SecondClosestCommentLines)["nonNamespaced"] == "true")
+	namespaced := (types.ExtractCommentTags("+", t.SecondClosestCommentLines)["nonNamespaced"] == nil)
```

Comment:
At some point we need to convert this to a helper method `types.HasComment(...)`